samples/README.md

```diff
@@ -1,3 +1,8 @@
 # Zeebe Java Client Examples
 
-This Maven project contains a number of examples each performing a specific task using the Zeebe Java client. For an overview and context have a look at the Zeebe documentation: https://docs.zeebe.io/java-client-examples/README.html
\ No newline at end of file
+This Maven project contains a number of examples each performing a specific task using the Zeebe
+Java client.
+
+For an overview and context have a look at
+the [Zeebe documentation](https://docs.camunda.io/docs/product-manuals/clients/java-client-examples/index)
```

Comment:
This seems to be a dead link. I wonder whether index should be in the url

Reply:
It is rather a not yet alive link. It will work when we update the documentation.

And yes, index needs to be in the URL